.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,25 @@
+name: Bug report
+description: Something not working as expected? Let us look into it.
+labels: ["bug"]
+body:
+  - type: textarea
+    id: bug-description
+    attributes:
+      label: Bug Description
+      value: "## Bug description\n\n*Please describe.*\n*If this affects the front-end, screenshots would be of great help.*\n\n*If you are on PostHog Cloud it would be really valuable if you can share any links where the problem occurs. This speeds up our ability to troubleshoot tremendously.*\n\n## How to reproduce\n\n1. \n2.\n3. \n\n## Additional context\n\n"
+      render: markdown
+    validations:
+      required: true
+  
+  - type: textarea
+    id: debug-info
+    attributes:
+      label: Debug info
+      value: "- [ ] PostHog Cloud, Debug information: [please copy/paste from https://us.posthog.com/settings/project-details#variables]\n- [ ] PostHog Hobby self-hosted with `docker compose`, version/commit: [please provide]\n- [ ] PostHog self-hosted with Kubernetes (deprecated, see [`Sunsetting Kubernetes support`](https://posthog.com/blog/sunsetting-helm-support-posthog)), version/commit: [please provide]\n"
```

Comment:
This value will contain extra spaces.
![2024-04-12 at 02 24 25@2x](https://github.com/PostHog/posthog/assets/13001502/5c37df7a-5955-4dbd-b189-b80200eee439)